admin/python/lsst/qserv/testing/main.py

```diff
@@ -0,0 +1,146 @@
+"""Command line script for running KPM tests.
+
+To run it just make a small wrapper script (or use Python
+setup process to define an entry point).
+"""
+
+import argparse
+import functools
+import logging
+import os
+
+import MySQLdb
+
+from .config import Config
+from . import mock_db
+from .monitor import InfluxDBFileMonitor, LogMonitor
+from .runner_mgr import RunnerManager
+
+_LOG = logging.getLogger(__name__)
+
+
+def _logConfig(level, slot):
+    levels = {0: logging.WARNING, 1: logging.INFO, 2: logging.DEBUG}
+    if slot is None:
+        simple_format = "%(asctime)s %(levelname)7s %(name)s -- %(message)s"
+    else:
+        simple_format = f"%(asctime)s %(levelname)7s %(name)s [slot:{slot}] -- %(message)s"
+    level = levels.get(level, logging.DEBUG)
+    logging.basicConfig(format=simple_format, level=level)
```

Comment:
`basicConfig`'s `level` can take a string (tho you have to validate inputs or handle exception if it's wrong)

Reply:
Hmm, that looks like an undocumented feature.